src/widget/wtracktableview.h

```diff
@@ -39,8 +39,7 @@ class WTrackTableView : public WLibraryTableView {
     void resizeEvent(QResizeEvent* event) override;
     void activateSelectedTrack();
     void loadSelectedTrackToGroup(const QString& group, bool play);
-    void assignNextTrackColor() override;
-    void assignPreviousTrackColor() override;
+    void selectTrackColor(int) override;
```

Comment:
please give it a name so the we can easily see the purpose of the int.
```suggestion
    void selectTrackColor(int steps) override;
```